src/include/mpir_datatype.h

```diff
@@ -310,10 +310,12 @@ void MPIR_Datatype_free(MPIR_Datatype * ptr);
 
 /* we pessimistically assume that MPI_DATATYPE_NULL may be passed as a "valid" type
  * for send/recv when MPI_PROC_NULL is the destination/src */
+#define MPIR_Datatype_is_builtin(datatype_) HANDLE_IS_BUILTIN((datatype_))
```

Comment:
I'm not convinced adding this alias is worthwhile. Especially since it would only exist for datatypes. What do you think of just using `HANDLE_IS_BUILTIN()` everywhere?

Reply:
I am ok with either one. I will remove the alias.